### DISCUSSION TITLE
zipped/gzipped files containing more than one file

### DISCUSSION THREAD

**Discussion by nmayle** [CMSgov/price-transparency-guide#598] [Q&A]
If an in_network file location points to a zip or gzip file, is that file allowed to contain more than one file?  We are seeing zip or gzip files that contain directories with more than one file.  We have seen this for in_network files and also for provider_reference files.

Zipping together several files was discussed [#59](https://github.com/CMSgov/price-transparency-guide/discussions/59), but it wasn't clear if this was allowed. The specification for the table of contents seem to imply that the in_network file referenced by a location is a single file (with a single description) and compressing was just a means of reducing file size; it would seem odd if compressing allowed what should be a single in_network file to reference multiple files. Why would compressing be allowed to change the structure of the data?

Have others seen this issue?  Does it make any sense for the compressed file to expand into a directory of files?

**Comment by dgolden** [comment]
@nmayle  I have not seen this. Can you share the name of the Payer, or better yet, the URI? 
My understanding is that compression should be used only to reduce file size.

I'd suggest sending an email to  CMS, pointing out that this practice is inconsistent with their guidelines, and with other Payer practices, and that the CMS technical notes should be updated to expressly prohibit this practice.

**Comment by Devyandu** [comment]
Zipping multiple files into a single zip file is not allowed. It is mentioned as a comment in the same discussion thread you reference.

[From :Scott - Agreed, we are not combining multiple files into a single file -- this question is more about compressing multiple files into a single archive.](https://github.com/CMSgov/price-transparency-guide/discussions/59#discussioncomment-842601)

https://github.com/CMSgov/price-transparency-guide/discussions/59#discussioncomment-842601

**Comment by nmayle** [comment]
Humana is using zip files containing multiple in-network files.
Has anyone else noticed this?  

Has anyone reported this to them?  Is anyone from Humana reading this?
